src/inc/msquic.hpp

```diff
@@ -1552,3 +1557,5 @@ struct QuicBufferScope {
     operator QUIC_BUFFER* () noexcept { return Buffer; }
     ~QuicBufferScope() noexcept { if (Buffer) { delete[](uint8_t*) Buffer; } }
 };
+
+#endif
```

Comment:
Please add comments to indicate what this ends.
```suggestion
#endif // _WIN32
```

Reply:
Thanks for your review ~
I added that comment in a new commit.